src/template/wasm2c.declarations.c

```diff
@@ -89,20 +89,24 @@ static inline bool func_types_eq(const wasm_rt_func_type_t a,
   (CHECK_CALL_INDIRECT(table, ft, x),       \
    DO_CALL_INDIRECT(table, t, x, __VA_ARGS__))
 
-#ifdef SUPPORT_MEMORY64
+#if __has_builtin(__builtin_add_overflow)
+#define add_overflow(a, b, resptr) __builtin_add_overflow(a, b, resptr)
+#elif defined(_MSC_VER)
+static inline bool add_overflow(uint64_t a, uint64_t b, uint64_t* resptr) {
+  return _addcarry_u64(0, a, b, resptr);
+}
```

Comment:
Maybe used `#define` here for consistency with above?   Or use `static inline` function above?

Reply:
Done.